app/src/main/java/com/cornellappdev/score/components/SportCard.kt

```diff
@@ -138,10 +149,26 @@ fun SportCard(
                         style = universityText
                     )
                 }
-                Text(
-                    text = date,
-                    style = dateText,
-                )
+                val currentDate = SimpleDateFormat("yyyy-MM-dd", Locale.getDefault()).format(Date())
+
+                if (currentDate != date) {
+                    Text(
+                        text = date,
+                        style = dateText
+                    )
+                } else {
+                    Row( horizontalArrangement = Arrangement.spacedBy(4.dp),
+                        verticalAlignment = Alignment.CenterVertically,) {
+                        Box(
+                            modifier = Modifier
+                                .padding(1.dp)
+                                .width(8.dp)
+                                .height(8.dp)
+                                .background(saturatedGreen, shape = CircleShape)
```

Comment:
We should make this opacity fade like a sine wave to communicate it being live. You should be able to do this with some sort of `animateFloatAsState()` ... `rememberInfiniteTransition()`... something like that. It's a very common problem, so I'd suggest finding an online / ChatGPT soln for this.

To reiterate, we should animate the opacity of this background as a sine wave. You can then do `saturatedGreen.copy(alpha = ...)` to input the alpha / opacity.